redbot/cogs/trivia/data/lists/2020.yaml

```diff
@@ -0,0 +1,97 @@
+AUTHOR: OofChair#0001
+When did Prince Harry and Meghan Markle leave the royal family?:
+- Jan 8
+- January 8
+- Jan 8th
+- January 8th
+When did COVID-19 first appear?:
```

Comment:
I'm guessing you meant the first appearance outside China?

Reply:
First appearance inside China

Reply:
Then I don't think that's true, [according to WHO](https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200423-sitrep-94-covid-19.pdf) the first human cases were reported in early December 2019.

Reply:
Ah yes, I am seeing that one of the earliest cases was in December 2019.